src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,298 @@
 #include <ripple/app/reporting/P2pProxy.h>
 #include <ripple/json/json_value.h>
 #include <ripple/net/RPCErr.h>
+#include <ripple/protocol/LedgerFormats.h>
+#include <ripple/protocol/SField.h>
+#include <ripple/protocol/TER.h>
+#include <ripple/protocol/TxFormats.h>
+#include <ripple/protocol/digest.h>
 #include <ripple/protocol/jss.h>
 #include <ripple/rpc/Context.h>
 #include <ripple/rpc/Role.h>
-#include <ripple/rpc/impl/TransactionSign.h>
```

Comment:
Add blank lines between the include groups. I.e. no lines between the ripple includes, then a blank line then the boot includes, then a blank line, then the standard includes.

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/ebe4258ac26d67cb544adfd7c7d693a3d72ce08e